incubator/starter/stack.yaml

```diff
@@ -1,6 +1,6 @@
-name: starter sample runnable stack
+name: Starter Sample
 version: 0.1.0
```

Comment:
We need to update the version before the release. Rest looks good and is ready to be merged/released.

Reply:
done